redis/pool.go

```diff
@@ -24,6 +24,8 @@ import (
 	"sync"
 	"sync/atomic"
 	"time"
+
+	"golang.org/x/net/context"
```

Comment:
above go 1.8, `golang.org/x/net/context` uses type alias to standard `context` package.
so it works well in all cases.

Reply:
Hi, does this mean extra dependency anyway?

Reply:
it's an empty package for above go1.9 https://github.com/golang/net/blob/master/context/go19.go

Reply:
removed on 21cf364